go.sum

```diff
@@ -22,6 +22,8 @@ github.com/adrg/xdg v0.3.2 h1:GUSGQ5pHdev83AYhDSS1A/CX+0JIsxbiWtow2DSA+RU=
 github.com/adrg/xdg v0.3.2/go.mod h1:7I2hH/IT30IsupOpKZ5ue7/qNi3CoKzD6tL3HwpaRMQ=
 github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc/go.mod h1:LOuyumcjzFXgccqObfd/Ljyb9UuFJ6TxHnclSeseNhc=
 github.com/alecthomas/units v0.0.0-20151022065526-2efee857e7cf/go.mod h1:ybxpYRFXyAe+OPACYpWeL0wqObRcbAqCMya13uyzqw0=
+github.com/ansel1/merry v1.5.0 h1:9E6Jj5Yda9SfY3dhbOZUq1CeHl5JA1+mItYvKqokJ7s=
+github.com/ansel1/merry v1.5.0/go.mod h1:wUy/yW0JX0ix9GYvUbciq+bi3jW/vlKPlbpI7qdZpOw=
```

Comment:
Hmmm... Unsure if there is something wrong with the go.sum, since idk how it works in practise. There seems to be too many changes IMO.

Reply:
these are most likely flume dependencies, so probably works as intended. if you want, you can revalidate this by reverting (your local) copy of go.mod and go.sum to what the parent was and then running `go build` (or practically any other relevant command, see [here](https://golang.org/ref/mod#go-mod-file-updates)).